x-pack/test/functional/services/ml/data_visualizer_table.ts

```diff
@@ -320,5 +323,17 @@ export function MachineLearningDataVisualizerTableProvider(
         await this.assertTextFieldContents(fieldName, docCountFormatted, exampleCount);
       }
     }
+
+    public async ensureNumRowsPerPage(n: 10 | 25 | 100) {
+      await retry.tryForTime(10000, async () => {
+        await testSubjects.existOrFail('tablePaginationPopoverButton');
+        await testSubjects.click('tablePaginationPopoverButton');
+        await testSubjects.click(`tablePagination-${n}-rows`);
+
+        expect(await testSubjects.getVisibleText('tablePaginationPopoverButton')).to.contain(
+          n.toString()
```

Comment:
This `contain` could potentially give a false positive here. E.g. if we're expecting `10` but the actual value is `100`, this validation will still succeed. Maybe it'd be better to take the full text, split on `:` and compare with `eql`?